server/routes/templates/infoForCreatedSuperadmin.hjs

```diff
@@ -38,12 +38,87 @@
   </style>
 </head>
 <body>
-    <div>
-        {{firstName}}
-    </div>
-    <div>
-        {{lastName}}
+    <div class="mail__wrapper max-w-md mx-auto"
+      style="box-sizing: inherit;border-width: 0;border-style: solid;border-color: #dae1e7;margin-left: auto;margin-right: auto;max-width: 40rem;">
+
+      <div class="mail__content bg-white p-8 shadow-md"
+        style="box-sizing: inherit;border-width: 0;border-style: solid;border-color: #dae1e7;background-color: #fff;padding: 2rem;box-shadow: 0 4px 8px 0 rgba(0,0,0,.12),0 2px 4px 0 rgba(0,0,0,.08);">
+
+        <div class="content__header text-center tracking-wide border-b"
+          style="box-sizing: inherit;border-width: 0;border-style: solid;border-color: #dae1e7;border-bottom-width: 1px;text-align: center;letter-spacing: .05em;">
+          <div class="text-red text-sm font-bold h-24"
+            style="box-sizing: inherit;border-width: 0;border-style: solid;border-color: #dae1e7;font-weight: 700;height: 6rem;color: #e3342f;font-size: 12px; font-family: Arial, Helvetica, sans-serif;">
+            <img src="http://app-production.eu:8080/assets/images/brand-logo.png"
+              style="width: 280px;box-sizing: inherit;border-width: 0;border-style: solid;border-color: #dae1e7;max-width: 100%;height: auto;">
+          </div>
+        </div>
+
+        <div class="content__body py-8 border-b"
+          style="box-sizing: inherit;border-width: 0;border-style: solid;border-color: #dae1e7;border-bottom-width: 1px;padding-top: 2rem;padding-bottom: 2rem;">
+          <p style="box-sizing: inherit;border-width: 0;border-style: solid;border-color: #dae1e7;margin: 0;">
+            Dear {{firstName}}, <br>
```

Comment:
All this need to be translated.
For this, we have fields like initialGreetings, etc.
See how it's looks for other emails.